docs/CODE_GUIDELINES.md

```diff
@@ -291,23 +291,25 @@ The design system's component directory contains all of the design system's comp
 
 ### Imports
 
-The framework follows a specific ordering/clustering for importing modules into a component. This is enforced through the [`import/order` lint rule](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md).
+There's a specific ordering/clustering the framework follows for importing modules into a component.
```

Comment:
could you keep the original changes in this `Imports` section? I changed it when moving things into the repo so copy/paste probably overrode it

Reply:
I reverted the Imports section